engine/src/main/scala/cromwell/webservice/metadata/MetadataBuilderActor.scala

```diff
@@ -214,6 +214,7 @@ class MetadataBuilderActor(serviceRegistryActor: ActorRef) extends LoggingFSM[Me
       context.parent ! RequestComplete(StatusCodes.OK, processMetadataResponse(query, metadata))
       allDone
     case Event(StatusLookupResponse(w, status), _) =>
+      // TODO PBE refactor this to use workflowMetadataResponse with includeCallsIfEmpty = false
```

Comment:
er actually don't do this, status goes against the summary table and not the journal